.github/workflows/ci.yml

```diff
@@ -32,22 +32,6 @@ jobs:
       - name: Lint
         run: pnpm run lint
 
-  typecheck:
```

Comment:
What do you mean? We had a typecheck test for a long time now, every time there is a type error, every CI job fails

Reply:
Some type error causes rollup dts build to fail, so essentially breaks all CI jobs. But I think I saw some cases where only root `tsc` catches the error.

`vitest --typecheck` inside `test/...` directory doesn't do the same thing, right?

Reply:
`vitest --typecheck` inside `test/...` should catch type errors as "Source Errors" if they happen to be outside the test directory

Reply:
Ah, really? What I meant was running `vitest --typecheck` inside `test/config` (`@vitest/test-config` workspace). Does it do equivalent of `tsc -p tsconfig.check.json` at the root?

Reply:
No, it uses a different `tsconfig`. The root tsconfig is a bit weird as it tests too much and too little at the same time (doesn't test with the output d.ts, but internal types, and doesn't test ui)

Reply:
Yeah, I don't really understand what `tsconfig.check.json` doing exactly, but I just wanted to check. You think we don't have to run that anymore on CI since each package's rollup build does enough typecheck?

Reply:
Yes, I think it's too much. We do probably need a typecheck step for the UI package though.